Bindings/Java/Matlab/Hopper_Device/OptimizeHopper.m

```diff
@@ -201,4 +182,57 @@
 
 end
 
+function plotSolution(x, problem)
+
+% Construct the actuator controls from solution 'x'. See 
+% CONSTRUCTCONTROLS above for details.
+[deviceControl, muscleExcitation, t] = constructControls(x, problem);
+                             
+% Plot optimized controls.
+figure;
+set(gcf, 'units', 'normalized', 'Position', [0.1 0.1 0.8 0.8])
+subplot(2,1,1);
+plot(deviceControl(1,:), deviceControl(2,:), 'b-', 'linewidth', 2)
+hold on
+plot(muscleExcitation(1,:), muscleExcitation(2,:), 'r-', 'linewidth', 2)
+ylim([-0.10 1.10]); xlim([0 5]); xticks(0:1:5);
+switch problem
+    case 'one_hop'
+        text_x = [x/2 t];
+        text_y = [1.05 -0.05];
+        labels = {'<--x(1)-->', 't1'};
```

Comment:
I wonder if the `<--x(1)-->` will consistently fill up the correct amount of space on different computers or for different figure sizes; perhaps there are cases where the `<...>` extend beyond the desired interval.